src/Attr.cc

```diff
@@ -359,8 +363,24 @@ void Attributes::CheckAttr(Attr* a)
 			}
 			break;
 
+		case ATTR_DEFAULT_INSERT:
+			{
+			if ( ! type->IsTable() )
+				{
+				Error("&default_insert only applicable to tables");
+				break;
+				}
+			}
+			// fallthrough
```

Comment:
I'd prefer copying the the call to `check_default_attr` here instead of the fall-through. The fall-through  makes code reading harder and is easy to miss when working on the code in the future (like adding another new attribute). (And seems the  check for having both attributes is fine to have at only one location.)